packages/cactus-plugin-persistence-fabric/src/main/typescript/web-services/status-endpoint-v1.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * OpenAPI endpoint (GET) for reading status of the persistence plugin.
+ */
+
+import {
+  Logger,
+  Checks,
+  LogLevelDesc,
+  LoggerProvider,
+  IAsyncProvider,
+} from "@hyperledger/cactus-common";
+import type {
+  IEndpointAuthzOptions,
+  IExpressRequestHandler,
+  IWebServiceEndpoint,
+} from "@hyperledger/cactus-core-api";
+import { registerWebServiceEndpoint } from "@hyperledger/cactus-core";
+
+import { PluginPersistenceFabric } from "../plugin-persistence-fabric";
+import OAS from "../../json/openapi.json";
+
+import type { Express, Request, Response } from "express";
+import safeStringify from "fast-safe-stringify";
+import sanitizeHtml from "sanitize-html";
+
+export interface IStatusEndpointV1Options {
+  logLevel?: LogLevelDesc;
+  connector: PluginPersistenceFabric;
+}
+
+/**
+ * OpenAPI endpoint (GET) for reading status of the persistence plugin.
+ */
+export class StatusEndpointV1 implements IWebServiceEndpoint {
+  public static readonly CLASS_NAME = "StatusEndpointV1";
+
+  private readonly log: Logger;
+
+  public get className(): string {
+    return StatusEndpointV1.CLASS_NAME;
+  }
+
+  constructor(public readonly options: IStatusEndpointV1Options) {
+    const fnTag = `${this.className}#constructor()`;
+    Checks.truthy(options, `${fnTag} arg options`);
+    Checks.truthy(options.connector, `${fnTag} arg options.connector`);
+
+    const level = this.options.logLevel || "INFO";
+    const label = this.className;
+    this.log = LoggerProvider.getOrCreate({ level, label });
+  }
+
+  public getOasPath(): any {
+    this.log.debug(OAS);
```

Comment:
Should we not return the OAS.path here (the commented code)?

Reply:
ok removing